plugins/lua/buildingplan/planneroverlay.lua

```diff
@@ -207,40 +207,47 @@ ItemLine.ATTRS{
 }
 
 function ItemLine:init()
-    self.frame.h = 1
+    --self.frame.h = 2
```

Comment:
this is subtle, but by commenting this out, then the bounding box for the ItemLine becomes unrestricted (within the bounds of its parent). The effect of this is that clicking anywhere on the panel registers as a click on the first visible ItemLine. try this: plan a well, hit `q`  a few times so a lower item is selected, then click on the second ItemLine text. The *first* ItemLine will register the click and steal the selection.

Reply:
a-ha! so _that's_ what that was!! I was sure i encountered a mis-behaviour like this, at some point!
yeah, most of these commented parts are probably leftovers of my experimentations towards using ResizingPanels for list entries... i'll have to take a look at all of these and fix 'em D: